Add Key Wheel Converter plugin

rdswift opened the pull request:
Based on the discussion in the [Community Forum](https://community.metabrainz.org/t/picard-acousticbrainz-music-key/579092) this plugin provides the ability to convert key information between 'camelot', 'open key' and 'standard' formats.
It adds three new scripting functions:

* **$key2camelot(key)** returns the key string `key` in camelot key format.
* **$key2openkey(key)** returns the key string `key` in open key format.
* **$key2standard(key\[,symbols\])** returns the key string `key` in standard key format.  If the optional argument `symbols` is set, then the '♭' and '#' symbols will be used, rather than spelling out '-Flat' and '-Sharp'.

The `key` argument can be entered in any of the supported formats, such as '2B' (camelot), '6d' (open key), 'A♭ Minor' (standard with symbols) or 'A-Flat Minor' (standard with text).  If the `key` argument is not recognized as one of the standard keys in the supported formats, then an empty string will be returned.

rdswift committed:
Add Key Wheel Converter plugin

phw reviewed (approved):
This is a great plugin addition. Thanks

phw commented on plugins/key_wheel_converter/key_wheel_converter.py:
```diff
@@ -0,0 +1,424 @@
+PLUGIN_DESCRIPTION = '''
+<p>
+Adds functions to convert between 'standard', 'camelot' and 'open key' key formats.
+</p>
```
Maybe a minor thing, but let's not add the <p> tags here. For one this does not really provide a value here with only a single paragraph. And also going forward I'd prefer to use Markdown (but that requires Picard >=2.7).

phw commented on plugins/key_wheel_converter/key_wheel_converter.py:
```diff
@@ -0,0 +1,424 @@
+
+    def __init__(self):
+        """Class to hold the mapping dictionary.
+        """
```
The constructor is not needed here, or is there some reason?

rdswift replied:
Force of habit.  Now removed.

rdswift committed:
General cleanup:

- Remove unneeded `__init__` and paragraph tags
- remove commented out line of code
- Avoid unnecessary assignment to `_return` variable
